### PR TITLE
Retarget 2021.1 EAP 2

### PR DESCRIPTION
**Pull request by maartenba**
Not sure about the removal of `javac2` path in `build.gradle` (seems fine, @sdubov, not sure if it was still needed...)

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle [hunk 1]
```diff
@@ -140,14 +140,6 @@ configure(mainProjects) {
         jvmArgs '-Xmx2048m'
     }
 
-    instrumentCode {
-        // javac2 is required to compile .form files from IntelliJ IDEA UI Designer
```
Comment (sdubov) [inline PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle:144]: Hm... Don't we still need this to compile Java `.forms` code? MS side is built on forms.

**Review by sdubov — APPROVED** [review]
Approved. Just one small question about `javac2.jar`.

File: PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle (hunk 1 above)
Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle:144]:
```
A problem was found with the configuration of task ':instrumentCode' (type 'IntelliJInstrumentCodeTask').
> File '....\.gradle\caches\modules-2\files-2.1\com.jetbrains.intellij.idea\ideaIC\211.4961.30-EAP-SNAPSHOT\cc7e56bf686f8783c0471f36d85083748d44b387\ideaIC-211.4961.30-EAP-SNAPSHOT\lib\javac2.jar' specified for property 'javac2' does not exist.
```

Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/build.gradle:144]: So not sure :(

**Pull request merged**